transaction_linux_test.go

```diff
@@ -0,0 +1,102 @@
+//go:build linux
+
+package pam
+
+import (
+	"testing"
+)
+
+func Test_LinuxError(t *testing.T) {
+	t.Parallel()
+	if !CheckPamHasStartConfdir() {
+		t.Skip("this requires PAM with Conf dir support")
+	}
+
+	statuses := map[string]error{
+		"bad_item":   ErrBadItem,
+		"conv_again": ErrConvAgain,
+		"incomplete": ErrIncomplete,
+	}
+
+	testError(t, statuses)
+}
+
+func TestFailure_001(t *testing.T) {
```

Comment:
All the `TestFailure` can use `t.Parallel()` I feel, no?

Reply:
Yeah, I will add that here. I also debated removing these tests entirely - IIRC I added them to boost the test coverage percentage, however they rely on an implementation detail of Linux-PAM, i.e. the underlying function check the transaction pointer for NULL. With OpenPAM these tests crash with SEGV.